docs/references/strategies/README.md

```diff
@@ -105,3 +103,39 @@
 - [carthage](platforms/ios/carthage.md)
 - [cocoapods](platforms/ios/cocoapods.md)
 - [swiftPM](platforms/ios/swift.md)
+
+## Strategies by type
+
+Languages supported by FOSSA CLI can have multiple strategies for detecting dependencies, one primary strategy that yields ideal results and zero or more fallback strategies. Within this list of strategies, we have the concept of *static* and *dynamic* strategies. Static strategies parse files to find a dependency graph (example: parse a `package-lock.json` file). Dynamic strategies are required when analyzing package managers that do not offer complete lockfiles, such as Gradle or Golang. Dynamic strategies require a working build environment to operate in.
+
+> If the FOSSA CLI uses a fallback strategy and does not detect ideal results, you will see a "Warning" in the output when running `fossa analyze`.
```

Comment:
[nit] In technical writing, prefer to describe facts instead of speaking to the user directly. 

Similarly, prefer to be explicit when a causal chain exists. "It used a fallback strategy, therefore its results are not ideal" is more explicit about the causal relationship than "it used a fallback strategy and its results were not ideal"- the latter leaves room for the interpretation that it is possible to have ideal results with a fallback strategy, and then implies that in such a case no warning will be emitted.
```suggestion
> If the FOSSA CLI is forced to utilize a fallback strategy, meaning it did not detect ideal results, a warning is emitted in the scan summary after running `fossa analyze`.
```